autoload/slime.vim

```diff
@@ -11,35 +11,35 @@ function! s:_EscapeText(text)
       let result = call(override_fn, [a:text])
     elseif exists("*" . escape_text_fn)
       let result = call(escape_text_fn, [a:text])
-    end
-  end
+    endif
+  endif
```

Comment:
I didn't realize the code was so inconsistent. I had to go check myself to believe it. (it happened over time… with many people hands' on the code)